crates/primitives/digest-log/src/lib.rs

```diff
@@ -57,10 +33,20 @@ pub enum FindLogError {
     MultipleLogs,
 }
 
-pub fn find_post_log(digest: &Digest) -> Result<PostLog, FindLogError> {
+pub fn find_starknet_block(digest: &Digest) -> Result<StarknetBlock, FindLogError> {
```

Comment:
add rust doc

Reply:
done

Reply:
does not match the format we use. missing # Arguments and # Return sections